RBQFetchedResultsController/Source/RBQFetchedResultsController.h

```diff
@@ -97,6 +97,20 @@
  */
 - (void)controllerDidChangeContent:(nonnull RBQFetchedResultsController *)controller;
 
+/**
+ *  This method is called before the controller performing fetch.
+ *
+ *  @param controller controller instance that performing fetch
+ */
+- (void)controllerWillPerformFetch:(nonnull RBQFetchedResultsController *)controller;
+
+/**
+ *  This method is called when the controller successfully fetching objects. It will not executed if the fetchRequest is nil.
```

Comment:
This method is called after the controller successfully fetches objects.

It will not be called if the `fetchRequest` is `nil`.